libs/core-ui/src/lib/util/datasetUtils/getColumnRanges.ts

```diff
@@ -20,7 +21,9 @@ export function getColumnRanges(
   const ranges = {};
   // get dataset features' range
   dataset.feature_names.forEach((feature) => {
-    const range = getDatasetFeatureRange(dataset, feature);
+    const range = ifEnableLargeData(dataset)
```

Comment:
Can we use tabular_dataset_metadata for both large data and non large data scenarios?

Reply:
I and Gaurav discussed this. We can use it, but Gaurav suggested using tabular_dataset_metadata for large data for now. tagging @gaugup

Reply:
Could you please a task for it to track, thank you!